build/deps/markdownlint.Dockerfile

```diff
@@ -1 +1 @@
-FROM ghcr.io/igorshubovych/markdownlint-cli:v0.32.0
+FROM ghcr.io/igorshubovych/markdownlint-cli:v0.31.1
```

Comment:
That seems to be a mis-merge. Please check other places too

Reply:
Fixed that.